packages/esm-appointments-app/src/routes.json

```diff
@@ -39,6 +39,66 @@
       "name": "todays-appointments-dashboard",
       "slot": "todays-appointment-slot",
       "component": "homeAppointments"
+    },
+    {
+      "name": "expected-appointments-panel",
+      "slot": "scheduled-appointments-panels-slot",
+      "component": "appointmentsByStatus",
+      "meta": {
+        "title": "expected",
+        "status": "Scheduled",
+        "showForPastDate": false,
+        "showForToday": true,
+        "showForFutureDate": true
+      }
+    },
+    {
+      "name": "checked-in-appointments-panel",
+      "slot": "scheduled-appointments-panels-slot",
+      "component": "appointmentsByStatus",
+      "meta": {
+        "title": "checkedIn",
+        "status": "CheckedIn",
+        "showForPastDate": false,
+        "showForToday": true,
+        "showForFutureDate": false
+      }
+    },
+    {
+      "name": "completed-appointments-panel",
+      "slot": "scheduled-appointments-panels-slot",
+      "component": "appointmentsByStatus",
+      "meta": {
+        "title": "completed",
+        "status": "Completed",
+        "showForPastDate": true,
+        "showForToday": true,
+        "showForFutureDate": false
+      }
+    },
+    {
+      "name": "missed-appointments-panel",
+      "slot": "scheduled-appointments-panels-slot",
+      "component": "appointmentsByStatus",
+      "meta": {
+        "title": "missed",
+        "status": "Missed",
+        "showForPastDate": true,
+        "showForToday": false,
+        "showForFutureDate": false
+      }
+    },
+    {
+      "name": "cancelled-appointments-panel",
+      "slot": "scheduled-appointments-panels-slot",
+      "component": "appointmentsByStatus",
+      "meta": {
+        "title": "cancelled",
+        "status": "Cancelled",
+        "showForPastDate": true,
+        "showForToday": true,
+        "showForFutureDate": true
+      }
```

Comment:
We should not wire in the "Early Appointments" component by default, because it requires the currently Palladium-custom "early appointments" endpoint, but you should be able to wire it in like this:

```
   {
      "name": "came-early-appointments-panel",
      "slot": "scheduled-appointments-panels-slot",
      "component": "earlyAppointments",
      "meta": {
        "title": "cameEarly",
        "showForPastDate": true,
        "showForToday": true,
        "showForFutureDate": true
      }
    }
```
... note that we don't need to set a "meta.status" here because the custom EarlyAppointments component handles getting the specific appointments required.  We really just need these meta parameters to set up the tabs properly.